.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 14
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - regression
+  - confirmed
+# Label to use when marking an issue as stale
+staleLabel: stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
```

Comment:
We should maybe mention somehow that labelling as confirmed stops stale bot for that specific PR/issue?

Reply:
I think we're the only ones who can label stuff, if I'm not mistaken, so that's not really a concern for the people who opened the issue or PR.